tests/acceptance/test_issue_details.py

```diff
@@ -83,17 +83,18 @@ def test_javascript_specific_event(self):
             platform='javascript'
         )
 
-        # Don't enable sentry10 so we have coverage for sentry9 as well.
```

Comment:
Shouldn't we keep this until we're ready to axe 9?

Reply:
We can hold off merging this until the next week or 2. It will be a dependency of https://github.com/getsentry/sentry/pull/13022 though